docker_entrypoint.sh

```diff
@@ -1,33 +1,62 @@
 #!/bin/bash
 
-set -e
+BITMONERO_DIR="/data/.bitmonero"
+MONERO_LOGS_DIR="$BITMONERO_DIR/logs"
+MONERO_WALLET_DIR="$BITMONERO_DIR/wallets"
+mkdir -p $MONERO_LOGS_DIR $MONERO_WALLET_DIR
+MONERO_LOG="$MONERO_LOGS_DIR/monerod.log"
+MONERO_WALLET_RPC_LOG="$MONERO_LOGS_DIR/monero-wallet-rpc.log"
+#monero.conf
+conf_template="/root/monero.conf.template"
+config_file="$BITMONERO_DIR/monero.conf"
+mv $conf_template $config_file
 
-BITMONERO_DIR=$(cat /root/monero.conf.template |grep "^data-dir="|sed "s/^data-dir=\([^ #]*\).*/\1/")
-cp /root/monero.conf.template $BITMONERO_DIR/
-new_conf_template="$BITMONERO_DIR/monero.conf.template"
-new_conf="$BITMONERO_DIR/monero.conf"
+#monero-wallet-rpc.conf
+wallet_rpc_conf_template="/root/monero-wallet-rpc.conf.template"
+wallet_rpc_conf="$BITMONERO_DIR/monero-wallet-rpc.conf"
+mv $wallet_rpc_conf_template $wallet_rpc_conf
 
-TOR_HOSTNAME=$(ip -4 route list match 0/0 | awk '{print $3}')
+MONERO_LAN_HOSTNAME="monerod.embassy"
+MONEROD_LOCAL_HOST="127.0.0.1"
+MONEROD_BIND_IP="0.0.0.0"
+TOR_SOCKS_PROXY_HOST=$(ip -4 route list match 0/0 | awk '{print $3}')
 TOR_PORT=9050
-STARTOS_PROXY_PORT=443
+STARTOS_REVPROXY_PORT=443
 MONERO_P2P_PORT=18080
-MONERO_RPC_PORT=18081
+MONERO_RPC_PORT=18089
```

Comment:
```suggestion
MONERO_RPC_PORT=18081
```
Mixed these up

Reply:
No, I (intentionally) put the unrestricted rpc on 127.0.0.1:18089.
The restricted one is on 0.0.0.0:18081 and called $MONERO_RPC_PORT_RESTRICTED

Reply:
Why move unrestricted away from 18081?